docs-chef-io/content/inspec/v6/install.md

```diff
@@ -0,0 +1,399 @@
++++
+title = "Install Chef InSpec 6"
+gh_repo = "inspec"
+
+## See the "content/inspec/install.md" file.
++++
+
+You can install Chef InSpec on MacOS, Windows, and Linux systems.
+
+## Install Chef InSpec 6
+
+You can download the latest Chef InSpec package relevant to your operating system
+from [our Downloads Page](https://www.chef.io/downloads).
+
+You can also install Chef InSpec using an installer, script, or package
+manager.
+
+### macOS
+
+#### Homebrew
+
+Chef InSpec is available as a standalone [Homebrew](https://brew.sh/) package.
+Run the following command in your terminal to install Chef InSpec:
+
+```bash
+brew install chef/chef/inspec
+```
+
+While this command is running, Homebrew may prompt you to enter your macOS user account and
+password to complete the installation.
+
+#### CLI
+
+You can install Chef InSpec using a curl script.
+
+```bash
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+```
+
+### Windows
+
+#### Installer
+
+Download a Windows Chef InSpec package from [Chef Downloads](https://www.chef.io/downloads),
+then double-click on the `.msi` file to launch the installer and follow the prompts.
+
+#### Powershell
+
+You can install Chef InSpec using the following Powershell script.
+
+```powershell
+. { iwr -useb https://omnitruck.chef.io/install.ps1 } | iex; install -project inspec
+```
+
+Once you have installed Chef InSpec, run `inspec version` to verify that the installation
+was successful.
+
+### Linux
+
+#### CLI
+
+The following curl script will install Chef InSpec for Ubuntu and Red Hat Enterprise Linux:
+
+```bash
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+```
+
+If you prefer, you can use a package manager to install Chef InSpec.
+Once you downloaded the latest [Chef InSpec package](https://www.chef.io/downloads)
+relevant to your Linux-based platform, use the command for the respective package
+manager listed below. Replace the example file path with the file path leading to
+your downloaded package.
+
+For Ubuntu, use the following command to install Chef InSpec:
+
+```bash
+sudo dpkg -i /path/to/inspec.deb
+```
+
+For Red Hat Enterprise Linux, use the following command to install Chef InSpec:
+
+```bash
+sudo rpm -U /path-to/inspec.rpm
+```
+
+For SUSE Linux Enterprise Server, use the following command to install Chef InSpec:
+
+```bash
+sudo zypper install /path-to/inspec.rpm
+```
+
+## Providing or Generating a License Key On First Run of Chef InSpec 6
```

Comment:
@IanMadd This section is new. Missing links. If you think it need restructuring I'm open to that.